context-free.md

```diff
@@ -2,21 +2,31 @@
 
 ##  Who is this programming language for?
 
+The programming language is for people that want to be able to create digital art without learning how to render it in a different programming language. In ContextFree, the only code that the user needs to write involves the image that they produce. The user does not need to learn about graphics libraries or additional features of a more complex language. In general, ContextFree is optimal for someone who wants to create graphical images who has a basic understanding of computer science.
```

Comment:
While I agree the programming language is for people that want to be able to create digital art without learning how to render it in a different programming language (I said this myself!), after the discussions we had in class, I realized you probably wouldn't want to use ContextFree for any non vector-based artwork because of the lack of pixel control. In this sense, after re-considering the question, I felt ContextFree users chose to use this specific programming language less because they didn't want to learn about graphics libraries or additional features of other languages and more because they had an idea of what they wanted to do (make fun recursive images) and ContextFree happened to work well.